dspace-server-webapp/src/test/resources/application-test.properties

```diff
@@ -14,4 +14,6 @@
 
 ## Log4j2 configuration for test environment
 ## This file is found on classpath at src/test/resources/log4j2-test.xml
-logging.config = classpath:log4j2-test.xml
\ No newline at end of file
+logging.config = classpath:log4j2-test.xml
+
+server.servlet.context-path=/
```

Comment:
Yes, is not expected.

Since the `/server` is added to the path by the `application-server` we need to explicitly tell the embedded tomcat to load the test application on the simpler `/` path in order to get everything working as it was before.